### ISSUE TITLE
all "BIEN_taxonomy_*()" queries list class as "Equisetopsida"

### ISSUE THREAD

**Issue by gabehassler** [bmaitner/RBIEN#25]
I was trying to get taxonomic information about a large number (>3,000) of species across the entire phylogeny, and I noticed that no matter which species I entered into `BIEN_taxonomy_species(<species>)`, the resulting dataframe listed the class as `Equisetopsida`.
All other information appears to be correct.

Here's an example:

```
> x <- BIEN_taxonomy_species(c("Arabidopsis thaliana", "Ginkgo biloba", "Sequoiadendron giganteum"))
> x$class
[1] "Equisetopsida" "Equisetopsida" "Equisetopsida" "Equisetopsida"
```

This is a great resource btw. Thanks so much for developing and maintaining it!

**Comment by bmaitner** [comment]
Thanks for the note!  This issue is related to taxonomic lags.  The higher classification used by Tropicos (and the TNRS, hence BIEN) at the time of our last update followed Reveal and Chase 2009 in applying the Equisetopsida to all land plants, not just horsetails. Horsetails are now known to be nested in the ferns.  @ojalaquellueva will this issue be addressed in the forthcoming update to BIEN?

**Comment by ojalaquellueva** [comment]
@bmainter correct. The TNRS classification is taken straight from Tropicos, which follows APG III (Reveal and Chase 2009) in placing all land plants in class Equisetopsida. It will take some fiddling to change that as Tropicos still hasn't changed their higher taxonomic backbone. Most likely I'll make the World Flora Online (WFO) backbone the default higher classification and keep Tropicos as an alternative.
